### PR TITLE
[stable/prometheus-node-exporter]Add support for nodePort

### PR DESCRIPTION
**Pull request by rmakram-ims**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
 Adding support for nodePort to the prometheus-node-exporter chart.
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus-node-exporter/values.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ service:
   type: ClusterIP
   port: 9100
   targetPort: 9100
+  nodePort:
```
Comment (gregdferrell) [inline stable/prometheus-node-exporter/values.yaml:13]: does this need a "0" (without the quotes) as a default value for nodePort as kube-state-metrics has here: https://github.com/helm/charts/blob/master/stable/kube-state-metrics/values.yaml?

Comment (rmakram-ims) [reply stable/prometheus-node-exporter/values.yaml:13]: there is no need to put a default value as there is a guard on the usage of the nodePort value.

**Comment by bamb00** [comment]
Does this need to be tested? I created a NodePort for node-exporter and try to access the endpoint with http://node_ip:nodeport_ip

      clusterIP: ""

      ## List of IP addresses at which the node-exporter service is available
      ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips
      ##
      externalIPs: []

      hostPort: 9100
      loadBalancerIP: ""
      loadBalancerSourceRanges: []
      servicePort: 9100
      type: NodePort
      nodePort: 31997

**Pull request merged**